_includes/imaging-tutorial-etcher-banner.html

```diff
@@ -1,5 +1,6 @@
 {% include icon.html type="info" %}
 There's an easier way to image your SD card! Check out
 [the "Getting Started" guide](/docs/getting-started#step-2-copy-the-image-on-to-the-sd-card){: .alert-link }
-for a cross-platform guide.
+for a cross-platform guide. This tutorial isn't being maintained and may not
```

Comment:
OK, I'll update it to include instructions to download the release from GitHub and assume there's a .zip download available.